src/Bridge/Laravel/Console/Commands/CreateProviderCommand.php

```diff
@@ -28,7 +28,7 @@ public function __construct()
      */
     public function handle(ProviderServiceInterface $providerService): void
     {
-        [$identifier, $name] = [$this->argument('identifier'), $this->argument('name')];
+        [$identifier, $name] = [$this->option('identifier'), $this->option('name')];
```

Comment:
You will need to use a method to check for arrays since the underlying typehint can return an array:

```
/**
     * Get option as a string or null
     *
     * @param string $key The option to get
     *
     * @return string|null
     */
    private function getOption(string $key): ?string
    {
        $option = $this->option($key) ?: null;

        // If option is an array reset
        return \is_array($option) ? \reset($option) : $option;
    }```

Reply:
@sjdaws is the `is_string` check @ https://github.com/loyaltycorp/multitenancy/pull/28/files#diff-ca0a1099a231a568b0202f331fa0f635R33 insufficient?